clients/android/NewsBlur/assets/dark_reading.css

```diff
@@ -1,22 +1,24 @@
-body, span {
+body, span, table, td, p, div {
 	background-color: #1A1A1A !important;
 	color: #FFF !important;
 }
 
 a, a span {
 	color: #319DC5 !important;
+	background-color: #1A1A1A !important;
 }
 
 a:visited, a:visited span {
 	color: #319DC5 !important;
+	background-color: #1A1A1A !important;
 }
 
-code {
+pre, blockquote, code {
     background-color: #4C4C4C;
 }
 
-pre, blockquote {
-    background-color: #4C4C4C;
+blockquote > p, pre > span, code span {
```

Comment:
http://www.nytimes.com/newsletters/what-were-reading?nlid=72249677 - still
needs a few tweaks. Title text is still black for example.

http://softwareleadweekly.com/

Pocket hits - not sure there is a link without being signed in

http://www.mountaingoatsoftware.com/ - weekly e-mail tips

http://www.longreads.com/ - weekly e-mail. This is the one the had inline
!important do doesn't look good at all for now.

On Mon, 7 Mar 2016 20:23 Samuel Clay, <notifications@github.com> wrote:

> In clients/android/NewsBlur/assets/dark_reading.css
> <https://github.com/samuelclay/NewsBlur/pull/892#discussion_r55265947>:
>
> >      background-color: #4C4C4C;
> >  }
> >
> > -pre, blockquote {
> > -    background-color: #4C4C4C;
> > +blockquote > p, pre > span, code span {
>
> Can you give me examples of the newsletters fixed by these selectors? I
> need to also implement this on iOS and possibly web.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/samuelclay/NewsBlur/pull/892/files#r55265947>.
>